eo-maven-plugin/src/main/java/org/eolang/maven/rust/PrimeModule.java

```diff
@@ -58,4 +58,32 @@ public PrimeModule(final String method, final String file) {
             file
         );
     }
+
+    /**
+     * Translates ("mangles") Java method name to native method name.
+     * @param jname Name of this function (and class) in Java.
+     * @return How it named in Rust.
+     */
+    private static String translate(final String jname) {
+        return PrimeModule.translate(
+            "EOrust.natives.".concat(jname),
+            jname
+        );
+    }
+
+    /**
+     * Translates ("mangles") Java method name to native method name according to
+     * <a href="https://docs.oracle.com/javase/1.5.0/docs/guide/jni/spec/design.html">documentation</a>
+     * or <a href="https://stackoverflow.com/questions/32470463/what-is-the-naming-convention-for-java-native-interface-method-and-module-name">stackoverflow</a>.
+     * @param clazz Simple name of class.
+     * @param method Name of method.
+     * @return How it named in Rust.
+     */
+    private static String translate(final String clazz, final String method) {
+        return String.format(
+            "Java_%s_%s",
+            clazz.replace("_", "_1").replace(".", "_"),
```

Comment:
@levBagryansky I didn't get what is `"_1"` for

Reply:
@maxonfjvipon There are 2 refs in commentary: https://stackoverflow.com/questions/32470463/what-is-the-naming-convention-for-java-native-interface-method-and-module-name">stackoverflow
Or, according to oracle spec:
|  Escape Sequence |  Denotes  |
|---|---|
|  _1  |  the character “_”  | 
| _2 | the character “;” in signature |
| _3 | 	the character “[“ in signatures |
In our case we have "_" only